django/db/models/options.py

```diff
@@ -387,7 +392,7 @@ def managers(self):
 
     @cached_property
     def managers_map(self):
-        return {manager.name: manager for manager in reversed(self.managers)}
+        return {manager.name: manager for manager in self.managers}
```

Comment:
`reversed` was there to deal with duplicates in `managers` so that the child model wins over a parent model in case of overridden managers. Since duplicates are no longer possible it's not needed. I've added an assertion to the test case just in case.